strawberryfields/program.py

```diff
@@ -605,6 +605,8 @@ def compile(self, *, device=None, compiler=None, **kwargs):
                 The default is False.
             warn_connected (bool): If True, the user is warned if the quantum circuit is not weakly
                 connected. The default is True.
+            shots (int): Number of times the program measurement evaluation is repeated. Passed
+                along to the compiled program's `run_options`.
```

Comment:
I think this is rendered in RST:
```suggestion
                along to the compiled program's ``run_options``.
```

Reply:
Good catch!